docker-compose.yml

```diff
@@ -22,5 +22,33 @@ services:
       - LOGBACK_REQUIRE_THREAD
       - LOGBACK_REQUIRE_ALERT_LEVEL=false
       - LOGBACK_REQUIRE_ERROR_CODE=false
+      - api_DB_HOST=darts-api-database
+      - api_DB_USERNAME=std_user
+      - api_DB_PASSWORD=std_pass
     ports:
       - $SERVER_PORT:$SERVER_PORT
+    depends_on:
+      - darts-api-database
+    links:
+      - darts-api-database
+
```

Comment:
I could be wrong on this but wondering if we need both depends_on and links?  I'd thought that in DC 2.1 we just needed depends_on and the default docker network would allow the services to talk.